go.sum

```diff
@@ -1226,6 +1226,8 @@ github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15 h1:hqAk8riJvK4RM
 github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15/go.mod h1:z4YtwM70uOnk8h0pjJYlj3zdYwi9l03By6iAIF5j/Pk=
 github.com/tendermint/flutter v1.0.0 h1:z3dKFopmX8OqCBdclrnziZHTWmaKqmSfmx/5nB8yQR0=
 github.com/tendermint/flutter v1.0.0/go.mod h1:hW8URRpeA0KEaBWl9sOJUAlFLXeUrNWMvUoahMi1s3A=
```

Comment:
you should run `go mod tidy` to remove the old references from the flutter `v1.0.0` from the `go.sum`

Reply:
done